org.gramps_project.Gramps.yml

```diff
@@ -0,0 +1,118 @@
+app-id: org.gramps_project.Gramps
+# flatpak-builder will not take com.gramps-project.Gramps
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: /app/bin/gramps
+rename-icon: gramps
+finish-args:
+# needs home access to access documents and pictures
+  - --filesystem=home
+# needs to own upstream name
+  - --own-name=org.gramps-project.Gramps
+# for gui
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+#needs network access for maps
+  - --share=network
+modules:
+  - name: BSDDB3
+    subdir: dist
+    builddir: true
+    config-opts:
+      - --enable-compat185
+      - --enable-shared
+      - --enable-cxx
+      - --enable-dbm
+      - --enable-stl
+    sources:
+      - type: archive
+        url: https://download.oracle.com/berkeley-db/db-5.3.28.tar.gz
+        sha256: e0a992d740709892e81f9d93f06daf305cf73fb81b545afe72478043172c3628
+      - type: patch
+        path: atomic.patch
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} .
+
+  - name: python3-bsddb
+    buildsystem: simple
+    build-options:
+      env:
+        BERKELEYDB_DIR: /app
+    build-commands:
+      - python3 setup.py build
+      - python3 setup.py install --prefix=/app
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/source/b/bsddb3/bsddb3-6.2.7.tar.gz
+        sha256: b0f7fa63eb240cd5815809c9d1d7df9f7cc8d6fa9619d0edbe9c794afc02dc9f
+
+  - name: osmgpsmapDependency
+    buildsystem: autotools
+    sources:
+       - type: archive
+         url:  https://github.com/nzjrs/osm-gps-map/releases/download/1.1.0/osm-gps-map-1.1.0.tar.gz
+         sha256:  8f2ff865ed9ed9786cc5373c37b341b876958416139d0065ebb785cf88d33586
+
+  - name: exiv2Dependency
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url:  https://www.exiv2.org/builds/exiv2-0.27.3-Source.tar.gz
+        sha256:  a79f5613812aa21755d578a297874fb59a85101e793edc64ec2c6bd994e3e778
```

Comment:
Thank you.  I copied and pasted the patches into my github as a test and added them to the manifest.  The url of exiv2 we use looks the same to me.  However, using the patches breaks the build, and I don't understand patches well enough to modify them myself.  How important is it for arch64 to be supported?  

[https://github.com/OzarkShepherd/flatpak/runs/1614006445?check_suite_focus=true]
> Building module exiv2Dependency in /home/runner/work/flatpak/flatpak/.flatpak-builder/build/exiv2Dependency-1
> ========================================================================
> Applying patch exiv2-arch-flags.patch
> patching file cmake/compilerFlags.cmake
> Hunk #1 FAILED at 26 (different line endings).
> 1 out of 1 hunk FAILED -- saving rejects to file cmake/compilerFlags.cmake.rej
> Error: module exiv2Dependency: Child process exited with code 1
> Error: Process completed with exit code 1.

Reply:
I tried it locally here and it did work.

Reply:
The patches you added don't apply because they are not the same as the one I linked to. Probably because the end of lines got changed.

And yes ARM 64 is very important.

Reply:
I removed the exiv2 and gexiv dependencies since they are optional and I couldn't figure out how to get them aarch64 compliant.

Reply:
Making sure the patches are not changed by the process to add the patches to the repository. I had no problem.

They where failing to apply because the end of lines changed between the original an the one in the repository.

Reply:
How did you make sure that the end of lines did not change between the original and the new file in the repository?  I'm using OpenSUSE with Gnome, and copied the contents of your original for Darktable into a new file at github in my repository.  It looked identical and had the same number of lines, but the file size was less than 10 bytes off and it got an error.  Then I tried copying from RAW format, but the paste still changed the line endings and file size was off slightly and it got an error.  Then I manually changed the line endings to carriage returns at the end of lines to see if that was the difference, but still got an error.  I don't really know what the difference in environment is that is causing the line endings to change in the patch, or what the original line endings are supposed to be.  Perhaps are you using a different OS than me?  Microsoft apparently uses different line endings than Linux.

Reply:
I usually use wget or curl directly to get the file. No text editor involved.

Reply:
Downloading the raw file in the web browser should work too (albeit I didn't check)

Reply:
Downloading the RAW file in the browser seems to have worked, thank you